docs/corpus.rst

```diff
@@ -145,6 +145,8 @@ a CLI tool that create the manifests given a corpus directory.
     - :func:`lhotse.recipes.prepare_librittsr`
   * - LJ Speech
     - :func:`lhotse.recipes.prepare_ljspeech`
+  * - MDCC
+    - :func:`lhotse.recipes.prepare_MDCC`
```

Comment:
Could you lowercase the function names (`prepare_MDCC` -> `prepare_mdcc`, etc.) for consistency with other recipes?

Reply:
Function names updated, thanks!